website/server/controllers/api-v3/members.js

```diff
@@ -402,6 +402,71 @@ function _getMembersForItem (type) {
   };
 }
 
+async function _handleGetMembersForChallenge (req, res) {
```

Comment:
In the first commit I just copied all the members-related content from the handleGetMembersForItem (:285) without changing anything inside this function for readability purposes, but considering how small the change actually is it might be a better idea to squash this commit with the next one. What do ya think?

Reply:
Don't worry about the commits, we squash them anyway when merging the pr